docs/index.md

```diff
@@ -1,12 +1,5 @@
-# ScalarDB
```

Comment:
@josh-wong @feeblefakie This `docs/index.md` is currently being published as GitHub pages:
https://scalar-labs.github.io/scalardb/

So, should we stop publishing it as GitHub pages, since this doc will be dedicated to for the doc site?

Reply:
Ah, right. Let's stop publishing it as GitHub pages.

Reply:
Sure.

Reply:
Once we have the docs site officially published, yes, I think we should stop publishing the docs in this repo through GitHub Pages. 

Since users might have the URL for this GitHub Pages site saved, we should consider having a basic way to direct users to the new docs site.

Reply:
So, probably, it's better to create a new file instead of updating index.md since it's not an index page any more.

Reply:
I've reverted the original `docs/index.md` (minus the library content) and created a new file that contains the library content (`docs/add-scalardb-to-your-build.md`). PTAL!